post/views.py

```diff
@@ -161,6 +161,34 @@ def delete(self, request, post_id):
 
 
 class CommentView(View):
+    def get(self, request, post_id):
+        try:
+            post = Post.objects.prefetch_related
+            ('comment_set', 'comment_set__author').get(id=post_id)
```

Comment:
```suggestion
            ('comment_set__author').get(id=post_id)
```
`comment_set__author` 하나로 `comment_set` 까지 가져오게 되어 앞부분은 불필요합니다.

Reply:
저러면 comment_set이랑 author가 둘다 자동으로 조인되는건가보네요? 감사합니다 수정했습니다